Update termux.md (#5286)
coder/code-server: docs/termux.md

# Termux

- [Install](#install)

> - [Direct Installation - Yarn Installation](yarn-installation)

Reviewer: ```suggestion
- [Direct Installation via yarn](#yarn-installation)
```

> Consider using a new user instead of root, read [here](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) why using root is not recommended.\
> Learn how to add a user [here](#create-a-new-user).

## Yarn Installation

> 1. Get [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.

Reviewer: ```suggestion
1. Install [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.
```

> 2. We will now change using the following command.

Reviewer: ```suggestion
2. Run the following command:
```

```sh
termux-change-repo
```
Now select `Main Repository` then change repo to `Mirrors by Grimler Hosted on grimler.se`.

3. After successfully updating of repository update and upgrade all the packages by the following command
```sh
pkg update
pkg upgrade -y
```

> 4. Now let's install requirement dependancy.

Reviewer: ```suggestion
4. Now let's install required dependencies:
```

```sh
pkg install -y \
  build-essential \
  binutils \
  pkg-config \
  python3 \
  yarn \
  nodejs-lts
npm config set python python3
node -v
```
you will get node version `v16.15.0`

5. Now install code-server
```sh
yarn global add code-server
```

> 6. Congratulation code-server is installed on your device using the following command.

Reviewer: ```suggestion
6. Congratulations! code-serve can now be installed on your device using the following command:
```

```sh
code-server --auth none
```

> 7. If already installed then use the following command for upgradation.

Reviewer: ```suggestion
7. To upgrade, run this command:
```
